modules/jdk/11/module.yaml

```diff
@@ -25,6 +25,7 @@ envs:
 packages:
   install:
   - java-11-openjdk-devel
+  - java-11-openjdk-jmods
```

Comment:
We don't want the jmods RPMs in the release builder images, please remove this line and the equivalent one in JDK17 module.